src/core/server/logging/layouts/conversions/date.ts

```diff
@@ -22,11 +22,14 @@ const formats = {
   UNIX_MILLIS: 'UNIX_MILLIS',
 };
 
-function formatDate(date: Date, dateFormat: string = formats.ISO8601, timezone?: string): string {
+function formatDate(
+  date: Date,
+  dateFormat: string = formats.ISO8601_TZ,
```

Comment:
Open question:
Do we consider changing the default from `UTC` to the hosts' timezone to be a breaking change?
If someone is ingesting logs generated by the Kibana Platform, they would need to modify their config to keep logs in `UTC`.

Reply:
Is this setting only used by the logging service? If so i dont think we should consider it a breaking change.

I've never seen a product that uses their log messages as an interfacing contract. If the setting is used in other places then maybe it should be considered a breaking change.

Reply:
@Bamieh AFAIK, `logging.timezone` is only used for logging.